### PR TITLE
created basic structure

### PR DESCRIPTION
**Pull request by Reggeditt**
#### Set up a new project with webpack based on existing template.
    [ ] Created an empty To Do List placeholder and set the index.html as a template using the HTML Webpack Plugin.
    [ ] Created an index.js file and set an array of some simple to do tasks. Each task object contains three keys:
             []description [string].
             []completed [bool].
             []index: [number].
    [ ] wrote a function to iterate over the tasks array and populate an HTML list item element for each task.
    [ ] Created a style.css file and set rules for the To Do List.

### CODE REVIEW

**Review by CollinsTatang — COMMENTED** [review]
Hi @Reggeditt, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that still needs an update on the README file.

- [ ] Please make sure you update all your main headers from the new [README template](https://github.com/microverseinc/readme-template).
- [ ] Please make sure that you add the section **Future Features**, including 1-3 potential features to be added.
- [ ] Please your commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). You can make the improvement in the next commit message.
- [ ] Kindly remove the microverse logo and replays it with yours if you have one.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi 🙋 @Reggeditt , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- PR is descriptive 👍 
- All Linter checks passed 👍 
- Tasks appear in order of index values for each task 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,43 +1,6 @@
...
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
-
-  <h3><b>Microverse README Template</b></h3>
-
-</div>
-
-<!-- TABLE OF CONTENTS -->
-
 # 📗 Table of Contents
```
[the remark this answers is not in the document]
Comment (rloterh) [reply README.md:7]: well resolved! ✔️

File: README.md [hunk 2]
```diff
@@ -60,34 +23,30 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [todo list app] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
```
[the remark this answers is not in the document]
Comment (rloterh) [reply README.md:65]: well resolved! ✔️

File: README.md [hunk 3]
```diff
@@ -97,25 +56,62 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[allows you to add tasks to a list]**
```
[the remark this answers is not in the document]
Comment (rloterh) [reply README.md:62]: well resolved! ✔️

File: README.md [hunk 4]
```diff
@@ -285,4 +274,4 @@ This project is [MIT](./LICENSE) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
[the remark this answers is not in the document]
Comment (rloterh) [reply README.md]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi TK,  

Your project is complete!🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
- Project is set up correctly!
- Good ReadMe!

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Reggeditt) [reply README.md]: thank you

**Pull request merged**